### PR TITLE
add NLP regression free training notebook

### PR DESCRIPTION
**Pull request by mansimov**
*Description of changes:* Add an example notebook for regression-free training for Natural Language Processing models

*Testing done:* Tested in the Sagemaker notebook studio. Please see the notebook logs with successful outputs

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb [hunk 1]
```diff
@@ -0,0 +1,2425 @@
...
+   "execution_count": null,
+   "id": "d4b3471b",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "conda_pytorch_p38",
```
Comment (YantaoShen) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb]: Hi Elman, you can change this line to '' "display_name": "Python 3 (PyTorch 1.8 Python 3.6 GPU Optimized)"

**Review by khetan2 — DISMISSED** [review]

**Review by YantaoShen — DISMISSED** [review]
LGTM

**Review by khetan2 — APPROVED** [review]
LGTM.

File: introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb [hunk 2]
```diff
@@ -0,0 +1,796 @@
+{
```
Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only notebook title (i.e. the very first title of this notebook) is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>Put all subsequent titles as H2 - only title is H1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_regression_free_training/Amazon_JumpStart_NLP_Regression_Free_Training.ipynb:1]: <p>No need to put heading for this sentence</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

**Review by atqy — APPROVED** [review]

**Pull request merged**